pkg/asset/cluster/tfvars.go

```diff
@@ -223,6 +223,7 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 				WorkerConfigs:               workerConfigs,
 				ImageURL:                    string(*rhcosImage),
 				PreexistingNetwork:          preexistingnetwork,
+				Private:                     installConfig.Config.ClusterPrivate(),
```

Comment:
pass in the installconfig's publishing strategy, because that gives tfvars function more control.

Reply:
We have a dilemma. In a previous iteration of this code, that's how I had it. Patrick (and I agree), believe publish should stay in the code so that it isn't necessary to handle in the terraform. So for now, until further discussed, I will keep this as is.

Reply:
>  In a previous iteration of this code, that's how I had it. Patrick (and I agree), believe publish should stay in the code so that it isn't necessary to handle in the terraform.

it's fine if you think it doesn't need to transfer to terraform, my comment was passing the publish strategy to TFVars function, so that it can decide to send private=true/false.. the decision doesn't have to be in the asset here... hopefully that clarifies it more.